sugoi-api-core/src/main/java/fr/insee/sugoi/core/service/OrganizationService.java

```diff
@@ -14,15 +14,19 @@
 package fr.insee.sugoi.core.service;
 
 import fr.insee.sugoi.core.model.PageResult;
+import fr.insee.sugoi.core.model.PageableResult;
 import fr.insee.sugoi.model.Organization;
 
 public interface OrganizationService {
 
-  Organization create(String realm, String storage, Organization organization);
+  Organization create(String realm, Organization organization, String storage);
```

Comment:
create(String realm, String storage, Organization organization) ? 

I find it weird to have organization between realm and storage.
Same things for all other methods.